src/main.cpp

```diff
@@ -56,7 +57,7 @@ int main(int argc, char* argv[])
             // type of loss/distance function to use
             case 'l':
                 loss = optarg;
-                if (loss.at(0) == 'L') {
+                if (std::regex_match(loss, std::regex("L\\d*"))) {
```

Comment:
This is great! But also needs to be changed here: https://github.com/ThrunGroup/BanditPAM/blob/main/src/kmedoids_ucb.cpp#L110

Reply:
On https://github.com/ThrunGroup/BanditPAM/blob/main/src/kmedoids_ucb.cpp#L110, the checking for loss starting with `L` here becomes redundant since the Lp norm is already checked in the `main.cpp` and the p is parsed and saved in `loss` variable.